rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -871,17 +872,17 @@ def testQuery_manyParticipants(self):
 
     self.submit_questionnaire_response(participant_id_1, questionnaire_id, RACE_WHITE_CODE, "male",
                                        "Bob", "Q", "Jones", "78751", "PIIState_VA",
-                                       "1234 Main Street", "Austin", "male_sex",
+                                       "1234 Main Street", "Austin", "male_sex", "512-555-5555",
```

Comment:
Let's pick a different phone number for login phone number, just to verify they are actually handled separately?